File: api/src/main/java/org/pmiops/workbench/Application.java
package org.pmiops.workbench;

import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.builder.SpringApplicationBuilder;

@SpringBootApplication
public class Application {

  public static void main(String[] args) {
    new SpringApplicationBuilder(Application.class).initializers().run(args);
  }
}

File: api/src/main/java/org/pmiops/workbench/Application.kt
package org.pmiops.workbench

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
object Application {

    @JvmStatic
    fun main(args: Array<String>) {
        SpringApplication.run(Application::class.java, *args)
    }
}
